source/installing-with-npm/index.html.md.erb

```diff
@@ -21,14 +21,9 @@ weight: 10
 
     If you're using Dart Sass 1.33.0 or greater, you may see deprecation warnings when compiling your Sass. You can [silence deprecation warnings caused by dependencies](/importing-css-assets-and-javascript/#silence-deprecation-warnings-from-dependencies-in-dart-sass) if required.
 
-    We recommend you do not use either LibSass or Ruby Sass, as they are deprecated.
+    We strongly recommend you do not use either LibSass or Ruby Sass, as they are deprecated.
```

Comment:
I think if we make this about 'new projects' (where there won't be an existing build pipeline etc) we can be even harder line on this.

```suggestion
    Do not use either LibSass or Ruby Sass for new projects, as they are deprecated.
```

Reply:
Really good shout. Suggesting a tweak to be super-clear about what's been deprecated.

```suggestion
    As LibSass and Ruby Sass are deprecated, do not use them for new projects.
```